### DISCUSSION TITLE
openclash可以启动YouTube访问超时

### DISCUSSION THREAD

**Discussion by ruikai0103** [vernesong/OpenClash#2710] [Q&A]
[root@OneCloud:~] # nslookup www.baidu.com
Server:         127.0.0.1
Address:        127.0.0.1#53

Name:      www.baidu.com
Address 1: 39.156.66.18
Address 2: 39.156.66.14
*** Can't find www.baidu.com: No answer

我的openclash可以运行成功，但是 YouTube访问是链接超时

路由器的IPV6的DHCP也关闭了， 只不过我的路由器链接的光猫  我看我的路由器上网信息  我是小米ax3600
连接类型：DHCP
IP地址：192.168.1.3
子网掩码：255.255.255.0
默认网关：192.168.1.1
DNS：192.168.1.1
IPv6连接类型：None

我运行  iptables -t nat -nL --line-number 返回

[root@OneCloud:~] # iptables -t nat -nL --line-number
Chain PREROUTING (policy ACCEPT)
num  target     prot opt source               destination
1    REDIRECT   tcp  --  0.0.0.0/0            8.8.4.4              redir ports 7892
2    REDIRECT   tcp  --  0.0.0.0/0            8.8.8.8              redir ports 7892
3    zone_wan_prerouting  all  --  0.0.0.0/0            0.0.0.0/0            /* !fw3 */
4    zone_tinc_prerouting  all  --  0.0.0.0/0            0.0.0.0/0            /* !fw3 */
5    openclash  tcp  --  0.0.0.0/0            0.0.0.0/0

应该是对着的吧

我的openclash 显示

IP 地址
IPIP      国内:查询中... 

IP.PC    国内:183.200.10.175 山西省 太原市 移通

IP.SB    海外:183.200.10.175 

IPIFY    海外:183.200.10.175 

网站访问检查
百度搜索:连接正常

网易云音乐:连接正常

GitHub:连接超时

YouTube:连接超时

**Comment by vernesong** [comment]
https://github.com/vernesong/OpenClash/wiki/%E7%BD%91%E7%BB%9C%E8%BF%9E%E6%8E%A5%E5%BC%82%E5%B8%B8%E6%97%B6%E6%8E%92%E6%9F%A5%E5%8E%9F%E5%9B%A0
